CaptureSDK/GiniCaptureSDK/Sources/GiniCaptureSDK/Networking/DocumentService.swift

```diff
@@ -51,7 +51,6 @@ public final class DocumentService: DocumentServiceProtocol {
                     guard errorOccurred == false else {
                         return
                     }
-                    errorOccurred = true
```

Comment:
[Suggestion] I think we need to keep the flag, why was it removed?

Reply:
The flag was the reason why the error screen was not triggered.